Documenting how to list 3rd party dependencies (#1319)
elastic/apm-agent-dotnet: CONTRIBUTING.md

### Coding guidelines

The repository contains a [.editorconfig file](.editorconfig) which is automatically picked up by Visual Studio and JetBrains Rider. Additionally, we also have a [.DotSettings file](ElasticApmAgent.sln.DotSettings). If you auto-format your code in Visual Studio or Rider, you will automatically confirm to our code styling. If you want to know more details about the specific conventions we use, feel free to look into the [.editorconfig file](.editorconfig).

> ### 3rd party dependencies

Reviewer: ```suggestion
### Third party dependencies
```

In case an agent project depends on a NuGet package, the dependency does not have to be listed, since it's not directly shipped with the agent. 

> However if source code from a 3rd party dependency is added to the agent, the following steps must be taken:

Reviewer: ```suggestion
However if source code from a third party dependency is added to the agent, the following steps must be taken:
```

> 1. Make sure the license of the 3rd party dependency allows us to take the code.

Reviewer: ```suggestion
1. Make sure the license of the third party dependency is one that is permitted to be included in the agent, based on Elastic's Open Source Policy
```
